### ISSUE TITLE
September 2018

### ISSUE THREAD

**Issue by jridgewell** [babel/proposals#48]
> PayPal: New York, NY (Sept 25-27)
> https://github.com/tc39/agendas/blob/master/2018/09.md

**Comment by jridgewell** [comment]
https://github.com/tc39/ecma262/pull/1250 achieved consensus.

**Comment by jridgewell** [comment]
https://github.com/tc39/ecma262/pull/1242 achieved consensus.

**Comment by jridgewell** [comment]
https://github.com/tc39/ecma262/pull/1288 needs an implementation before we're comfortable changing it.

**Comment by jridgewell** [comment]
https://github.com/bakkot/for-in-exploration to Stage 1.

**Comment by chicoxyzzy** [comment]
https://github.com/gibson042/ecma262-proposal-uniform-interchange-date-parsing to stage 1

**Comment by jridgewell** [comment]
https://github.com/jasonwilliams/proposal-promise-allSettled to stage 1

**Comment by jridgewell** [comment]
https://github.com/tc39/ecma262/pull/1174 needs an implementation. Babel?

**Comment by nicolo-ribaudo** [comment]
We already have the plug-in for that PR

**Comment by jridgewell** [comment]
> We already have the plug-in for that PR

Yes. I was trying to see if TC39 considered this sufficient to land the proposal, but they've decided it must be a native implementation. :sigh:

**Comment by jridgewell** [comment]
https://github.com/littledan/proposal-idl to stage 1

**Comment by jridgewell** [comment]
https://github.com/tc39/proposal-temporal to stage 2

**Comment by jridgewell** [comment]
https://github.com/gibson042/ecma262-proposal-JSON-parse-with-source to stage 1.
